src/components/App.tsx

```diff
@@ -24,15 +24,12 @@ import ToastContainer from './notifications/ToastContainer';
 import Notifier from './notifications/Notifier';
 import WorkspacesLayout from './workspace/WorkspacesLayout';
 import Graph from './graph/Graph';
-import WorkspaceLinkGetter from './WorkspaceLinkGetter';
 import MessagesViewConfigurator from './messages-view-configurator/MessagesViewConfigurator';
-import { useSearchStore } from '../hooks/useSearchStore';
-import { useTabsStore } from '../hooks';
+import { useSessionsStore } from '../hooks';
 import '../styles/root.scss';
 
 const AppRootBase = () => {
-	const searchStore = useSearchStore();
-	const tabsStore = useTabsStore();
+	const { sessionNames } = useSessionsStore();
```

Comment:
wait i think we confused something. sessionStore keeps sessions submitted by the user, and searchStore fetches all available sessions, and MessagesViewConfigurator waits for all sessions